test/support/test_utils.ex

```diff
@@ -99,10 +99,7 @@ defmodule Logflare.TestUtils do
     end)
   end
 
-  @doc """
-  gzipped request body from the Cloudflare Log Push HTTP service.
-  """
-
+  # gzipped request body from the Cloudflare Log Push HTTP service.
```

Comment:
Ah yeah saw your message in Slack. You should just be able to keep the first `@doc`.

Reply:
🆗 updated